src/Razor/src/Microsoft.NET.Sdk.Razor/Sdk/Sdk.Razor.StaticAssets.ProjectSystem.props

```diff
@@ -32,14 +31,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Content Include="**\*.config" ExcludeFromSingleFile="true" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
     <Content Include="**\*.json" ExcludeFromSingleFile="true" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
 
-    <!-- Set CopyToPublishDirectory to Never for items under AppDesignerFolder ("Properties", by default) to avoid publishing launchSettings.json -->
-    <Content Update="$(AppDesignerFolder)\**" CopyToPublishDirectory="Never" Condition="'$(AppDesignerFolder)' != ''"/>   
+    <!-- Set CopyToOutputDirectory & CopyToPublishDirectory to Never for items under AppDesignerFolder ("Properties", by default) to avoid copying launchSettings.json to the build or publish directory -->
```

Comment:
I thought there was some 'dotnet run' scenario that needed launchSettings.json in the bin folder.

Reply:
Do you recall what that might be? @davidfowl specifically made a change to not copy the launchsettings.json to the output directory here: https://github.com/aspnet/AspNetCore-Tooling/commit/63e8681fffcad8d577438af39af008068d2e7c37#diff-56a932829df1fcb51002be8ef7f7846dR26/

/cc @DamianEdwards

Reply:
I wanted to remove it from publish, I’m not sure if it’s needed in bin for dotnet run

Reply:
Ok. The file was already set to be removed from publish. I'll change it back to it's initial state.

Reply:
Once Bill is back, I will check with him if it needed in bin. For now, lets just remove from publish and not from bin.

Reply:
Yup, updated to do this. It's part of Content and it's only set to be removed from publish.